### ISSUE TITLE
The grid truncates the notes

### ISSUE THREAD

**Issue by 420tech** [MicrosoftDocs/windows-uwp#3987]
Formatting on the grid comparing is off. Last column has the notes and it is truncated.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 53f752d7-062a-4935-02c6-48371c208792
* Version Independent ID: daa135bb-7f77-3e62-05da-b18a9bfcdf12
* Content: [Data binding in depth - UWP applications](https://docs.microsoft.com/en-us/windows/uwp/data-binding/data-binding-in-depth#xbind-and-binding-feature-comparison)
* Content Source: [windows-apps-src/data-binding/data-binding-in-depth.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/data-binding/data-binding-in-depth.md)
* Product: **uwp**
* Technology: **data-and-files**
* GitHub Login: @alvinashcraft
* Microsoft Alias: **aashcraft**

**Comment by alvinashcraft** [comment]
I'll see if there's anything that can be done to adjust the grid layout. There is a horizontal scrollbar at the bottom of the grid to scroll right to see the rest of the Notes column.

**Comment by alvinashcraft** [comment]
Thanks for submitting your feedback. I went ahead and created & completed an internal work item to resolve this doc bug. I combined the 2nd and 3rd columns which makes the data fit within the available space without horizontal scrolling. The change has been merged and should be visible on the live Docs site soon. I will go ahead and close this item. 

Though the Issue will be marked as Closed, we will monitor any replies that come afterwards so treat this like an open Issue. Many thanks!
